Scripts/test-carthage-integration.sh

```diff
@@ -160,7 +160,7 @@ cp "../Demo/Configuration.swift" Source/Configuration.swift
 xcodegen generate
 
 echo_header "Run Tests"
-xcodebuild build test -project $PROJECT_NAME.xcodeproj -scheme App -destination "name=iPhone 15" CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO | xcpretty && exit ${PIPESTATUS[0]}
+xcodebuild build test -project $PROJECT_NAME.xcodeproj -scheme App -destination "name=iPhone 16" CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO | xcpretty && exit ${PIPESTATUS[0]}
```

Comment:
Would be nice to have it as parameter so we can update it without modifying script 🤔

Reply:
Agree - dunno why all of a sudden the iPhone 15 is not available anymore 🙃  (Will do so in a subsequent PR)